nomad/grapl-core.nomad

```diff
@@ -1069,4 +1070,48 @@ job "grapl-core" {
       }
     }
   }
+
+  group "plugin-registry" {
+    network {
+      mode = "bridge"
+
+      port "plugin-registry-port" {
+      }
+    }
+
+    task "plugin-registry" {
+      driver = "docker"
+
+      config {
+        image = var.container_images["plugin-registry"]
+        ports = ["plugin-registry-port"]
+      }
+
+      template {
+        data        = local.conditionally_defined_env_vars
+        destination = "plugin-registry.env"
+        env         = true
+      }
+
+      env {
+        PLUGIN_REGISTRY_BIND_ADDRESS    = "0.0.0.0:${NOMAD_PORT_plugin-registry-port}"
```

Comment:
would this ever _not_ be 0.0.0.0? if no, it's weird to make it part of the environment variable
but also: whatever, i don't care that much

Reply:
Seems unlikely it would change, but idk, doesn't seem worth changing.

Reply:
I think in general we should use _ADDRESS. The only reason I don't do that for the postgres connection is because the postgres connection string contains the username and password. While those are not meaningful access controls, it seems reasonable to at least treat the password as a secret where it's easy enough to do so.

Reply:
Yeah, that makes sense.